### ISSUE TITLE
The server disconnects everyone every 3 to 5 minutes when the timeout set to 360 or more

### ISSUE THREAD

**Issue by Ramon-Salazar** [BearWare/TeamTalk5#1970]
### Description

Sometimes have to adjust time out do to that in fact it should be better when the server has a very high user timeout, though it's this is a strange bug since everyone in the server are being disconnected, tested on many servers, even the last  beta v5.14.0.5125

### Application

- [ ] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [X] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

The timeout should be higher as usual, incase of reconnection of the same user

### Actual behavior

The server reconnects everyone every 3 to 5 minutes

### Steps to reproduce problem

going to server properties and adjusting the user time out, or by editing the xml file and seting the user time out

**Comment by amirmahdifard** [comment]
oh, i actually confurm this bug too! so now i wunder why everyone got disconnected every fue minits when i set user tymout to 960!

**Comment by Ramon-Salazar** [comment]
This bug existed for a very long time, I believe since v5.0 though strange it isn't reported for long time until now.

**Comment by amirmahdifard** [comment]
@Ramon-Salazar yeah i agree, i saw this bug from 5.7.1, but i wasn't sure if it is really the tymout problem to report, i just simply thort may be it's my internet problem that i'm getting disconnected rofl
